cost-model/src/transaction_cost.rs

```diff
@@ -98,8 +91,10 @@ pub struct UsageCostDetails {
     pub signature_cost: u64,
     pub write_lock_cost: u64,
     pub data_bytes_cost: u64,
-    pub builtins_execution_cost: u64,
-    pub bpf_execution_cost: u64,
+    // since https://github.com/solana-labs/solana/issues/30620 activated everywhere,
+    // all builtins consume compute units same as bpf. It is no longer necessary
+    // to separate builtin execution cost from bpf's
```

Comment:
I don't think we need this comment; since the code no longer separates the builtin from bpf I think this may only confuse the reader